exercises/strain/strain_test.exs

```diff
@@ -0,0 +1,95 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("strain.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule StrainTest do
+  use ExUnit.Case
+
+  defp is_odd(n), do: rem(n, 2) == 1
+  defp is_even(n), do: rem(n, 2) == 0
+
+  #@tag :pending
+  test "empty keep" do
+    assert Strain.keep([], fn e -> e < 10 end) == []
```

Comment:
What would you think about naming a `noop` function or something similar for these tests that are intended to be noops in the same way we've named `is_odd` and `is_even`?